h2o-py/h2o/frame.py

```diff
@@ -449,7 +451,7 @@ def _parse_raw(self, setup):
         p = {"destination_frame": None,
              "parse_type": None,
              "separator": None,
-             "single_quotes": None,
+             "single_quotes": setup["single_quotes"],
```

Comment:
Why is this necessary? Line 467 should already handle that

    p.update({k: v for k, v in viewitems(setup) if k in p})

Doesn't it?

Reply:
Yes this is redundant.

Reply:
reminder

Reply:
I guess I misunderstood this. This is to carry the results from parseSetup phase done by `ParseSetupHandler` to the actual parsing phase handled by ParseHandler.